pig/src/main/java/org/jboss/pnc/bacon/pig/impl/PigContext.java

```diff
@@ -83,6 +84,8 @@
 
     private String fullVersion; // version like 1.3.2.DR7
 
+    private Map<String, Collection<String>> checksums;
```

Comment:
(not for this commit) it would be good to have some more meaningful name. Sth like repositoryContentsChecksums